assets/javascripts/discourse/controllers/webinar-picker.js.es6

```diff
@@ -131,8 +131,8 @@ export default Controller.extend(ModalFunctionality, {
     return false;
   },
 
-  @discourseComputed("pastWebinarTitle", "pastStartDate")
-  pastWebinarDisabled(title, startDate) {
+  @discourseComputed
+  pastWebinarDisabled() {
```

Comment:
This depends on those two properties. The fix would be to use those function args instead. 🙂